scss/elements/badges.scss

```diff
@@ -0,0 +1,95 @@
+@mixin span-style($color, $background, $option, $width: 50%) {
+  position: absolute;
+  top: 0;
+  width: $width;
+  color: $color;
+  background-color: $background;
+
+  @if $option == left {
+    left: 0;
+  } @else if $option == right {
+    right: 0;
+  }
+}
+
+@mixin badge-style($color, $background, $is-splited: true) {
+  $box-shadow-first-two: 0 0.5em $background, 0 -0.5em $background;
+
+  @if $is-splited == true {
+    &:first-child {
+      @include span-style($color, $background, left);
+
+      box-shadow: $box-shadow-first-two, 0 0 $background, -0.5em 0 $background;
+    }
+    &:last-child {
+      @include span-style($color, $background, right);
+
+      box-shadow: $box-shadow-first-two, 0.5em 0 $background, 0 0 $background;
+    }
+  } @else {
+    &:first-child {
+      @include span-style($color, $background, 0, 100%);
+
+      box-shadow: $box-shadow-first-two, 0.5em 0 $background, -0.5em 0 $background;
+    }
+  }
+}
+
+@mixin setup($px) {
+  width: $px * 14;
+  height: $px;
+  font-size: $px * 1.2;
+  line-height: 1.5;
+}
+
+// Default style
+.nes-badge {
+  position: relative;
+  box-sizing: border-box;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  padding: 0.75em;
+  margin: 4px;
+  text-align: center;
+  white-space: nowrap;
+  vertical-align: middle;
+  cursor: $cursor-click-url;
```

Comment:
If you're doing a `:hover` style you should probably make sure to add a style for `:focus` as well

Reply:
My idea changed. 💡 
I'll write the details later. 📝

Reply:
If you want `nes-badge` to be clickable, I think that it is better to use `a` instead of `span`. like below. 🤔 
```html
<a href="#" class="nes-badge">
  <span>npm</span>
  <span>1.0.0</span>
</a>
```

so `cursor` and `:hover`, `:focus` style is unnecessary. 👍

Reply:
Hello!  Just discovered this fun project :) 

I think the opposite here: first, the markup should be correct (this is not a link!), then CSS is used to change the style.